components/main-with-sidebar/01-main-with-sidebar/readme.md

```diff
@@ -0,0 +1,17 @@
+A block with a main column and one sidebar on either the left or right (this example uses `mzp-l-sidebar-left`). A single page should only have one main section and one sidebar.
```

Comment:
We should mention in the notes that the outlines are just for demonstration, they're not part of the component. Lest someone assume it comes with colored borders.